### ISSUE TITLE
Allow customizing shortcuts

### ISSUE THREAD

**Issue by jjavierdguezas** [responsively-org/responsively-app#745]
# 🚀 Feature Request
Allow customizing shortcuts to avoid scenarios like in #643

**Comment by nikfp** [comment]
Has anyone started work on this issue yet? Or any of the upstream issues that could be blocking it? I'd like to know where to _try_ to help out.

**Comment by manojVivek** [comment]
@nikfp No, this is not being worked on. But if you're going to take this up, then I would suggest you implement this on the v1 app: https://github.com/responsively-org/responsively-app/tree/main/desktop-app-rewrite

**Comment by iamr-kumar** [comment]
@nikfp have you started working on it?

**Comment by nikfp** [comment]
@iamr-kumar as much as I'd like to say yes, life got in the way a bit. If you want to take it up feel free, you won't be duplicating any work from my end.

**Comment by Disaelfra** [comment]
@iamr-kumar are you working on this?

**Comment by bhumi-cka** [comment]
Is anyone working on this currently?

**Comment by manojVivek** [comment]
No, I don't think so.

**Comment by Disaelfra** [comment]
:wave: I'll take a pass it it.

**Comment by Bibhavshah** [comment]
Can anyone describe more about the issue, I am unclear what the guy wanted to do, it would surely like to try working on this issue.

**Comment by dssitachi** [comment]
Hello @manojVivek, can you please explain the issue or refer a link where I can find the necessary details. I would like to contribute on this

**Comment by Dailin521** [comment]
I checked the current desktop-app shortcuts implementation and I’m taking this up. My v1 scope is limited to the in-app shortcuts handled by the current shortcuts manager; I’m not planning to change Electron application menu accelerators unless maintainers want that broadened.
